hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java

```diff
@@ -193,11 +193,11 @@ public void run() {
 
         private void readFileStream(final InputStream tempStream) {
             String line;
+            final long totalNumberOfLines = getTotalNumberOfLines();
```

Comment:
Yes, the parameter type in `readEachLine` has allways been `long.` I don't know way it had been converted to double. Didn't make any sense to me.